### PR TITLE
Logged active user can add event

### PR DESCRIPTION
**Pull request by WojtekReu**
Still needs some switch in Event model: "it is event proposal, not accepted by redaction yet".

### CODE REVIEW

File: events/admin.py [hunk 1]
```diff
@@ -40,6 +41,8 @@ class EventAdmin(admin.ModelAdmin):
         'turnover',
         'absolute_price_change',
         'price_change',
+        'created_date',
+        'created_by',
```
Comment (JulianVolodia) [inline events/admin.py]: in Główne I think, also: (modification)
- is_moderated/activated column- every user submission should be reviewed?
- mod_date
- mod_by
- ...or even list of that changes? (new table in db)

File: events/forms.py [hunk 2]
```diff
@@ -18,3 +18,25 @@ def __init__(self, *args, **kwargs):
...
+
+
+class EventCreateForm(forms.ModelForm):
+    """
+    Adding new event by politikon users form
+    """
+    def save(self, commit=True):
+        """
+        Save event form to object
+        :param commit: If True, then the changes to ``instance`` will be saved to the database.
```
Comment (JulianVolodia) [inline events/forms.py]: it probably always should be, but every person should once per 30 sec for example (setting new db info: 'last_event_submit' date in user profile information)?

File: events/admin.py (hunk 1 above)
Comment (WojtekReu) [inline events/admin.py]: I've just added existed fields to visible list in Admin panel. I didn't create any new fields. Now user can add new event but can't modify it. Only staff can edit events (maybe even delete them).

File: events/forms.py (hunk 2 above)
Comment (WojtekReu) [inline events/forms.py]: It is default parameter exist for compatibility with others. Nothing change.

**Pull request merged**